packages/asli/asli.0.2.0/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+synopsis: "Interpreter for Arm's Architecture Specification Language (ASL)"
+description: """
+Interpreter for the language ASL that Arm uses to write their ISA specifications.
+
+Currently supports reading/typechecking the specification,
+interactive execution of ASL statements and expressions,
+executing opcodes one at a time,
+loading ELF files and executing Arm binaries."""
+maintainer: "Alastair Reid <alastair.d.reid@gmail.com>"
+authors: "Alastair Reid"
+license: "BSD-3-Clause"
+homepage: "https://github.com/alastairreid/asl-interpreter"
+bug-reports: "https://github.com/alastairreid/asl-interpreter/issues"
+depends: [
+  "dune" {>= "2.5"}
+  "ocaml" {>= "4.09" & build}
```

Comment:
The ocaml package generally should not be marked as a build dependency

Reply:
I updated the release with this change.
Is there any way that I can check these dependencies on my local machine before submitting?

(I also changed the build to only generate exe's (not byte code) to fix the z3 linking issue.)